### ISSUE TITLE
Label smoothing 적용 코드

### ISSUE THREAD

**Issue by JLake310** [boostcampaitech5/level2_klue-nlp-07#10]
```python
class CustomTrainer(Trainer):
    def compute_loss(self, model, inputs, return_outputs=False):
        labels = inputs.get("labels")
        outputs = model(**inputs)
        logits = outputs.get("logits")
        loss_fct = nn.CrossEntropyLoss(label_smoothing=0.1)
        loss = loss_fct(logits.view(-1, self.model.config.num_labels), labels.view(-1))
        return (loss, outputs) if return_outputs else loss
```
Trainer 클래스를 상속 받아 새로운 Trainer를 만들고,

```python
trainer = CustomTrainer(
        model=model,
        args=training_args,
	train_dataset=RE_train_dataset,
	eval_dataset=RE_dev_dataset,
	compute_metrics=compute_metrics
)

	
trainer.train()
model.save_pretrained('./best_model')
```

사용은 Trainer를 불러오는 부분만 수정하면 베이스라인과 동일하게 사용할 수 있습니다.
참고 자료는 [Transformers Trainer](https://huggingface.co/docs/transformers/v4.28.1/en/main_classes/trainer#transformers.TrainingArguments) 공식 document 입니다.
